LEAF_Nexus/db_mysql.php

```diff
@@ -55,62 +53,55 @@ public function __construct($host, $user, $pass, $database, $abortOnError = fals
                 $pass,
                 $pdo_options
             );
-
-            // make sure there are no active transactions on script termination
-            register_shutdown_function(function() {
```

Comment:
They were added in the other pr https://github.com/department-of-veterans-affairs/LEAF/pull/1755/files I do see where Jamie was coming from however I think this may need to go back in since it probably was there to take care of an issue with transactions.

Reply:
Like Shane said I just made the file exactly as it was before Leaf-3253 with the 2 exceptions that I stated in description to this PR